frontend/src/Metamaps/PasteInput.js

```diff
@@ -29,12 +28,10 @@ const PasteInput = {
       if (e.dataTransfer.files.length > 0) {
         self.handleFile(e.dataTransfer.files[0], coords)
       }
-      // OMG import bookmarks 😍
+      // OMG import bookmarks 😍 (Or just text :P)
```

Comment:
I'm pretty sure it's a paste event, but I'd have to verify with console.log or something to be 100% sure. Are you able to verify that and update the comment? We could even add a comment in the correct place saying "drag and drop text and it'll do this"